### PR TITLE
Add migration to add id properties to missing species details

### PR DESCRIPTION
**Pull request by lennym**
Finds the latest version of any new style project and iterates on it to add id properties to any species details sections that are missing them.

### CODE REVIEW

File: migrations/20200323140559_speciesdetails_ids.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          .catch(e => {
+            console.error(`Failed to update project ${project.id}`);
+            console.error(e.stack);
+          });
+      }, Promise.resolve());
+    });
+};
+
+exports.down = function(knex) {
+  // lolno
```
Comment (lennym) [inline migrations/20200323140559_speciesdetails_ids.js]: ¯\_(ツ)_/¯

Comment (joefitter) [reply migrations/20200323140559_speciesdetails_ids.js]: :D

**Comment by lennym** [comment]
Should be GTM now.

**Comment by joefitter** [comment]
👁

**Pull request merged**